javalib/src/main/scala/java/util/Formatter.scala

```diff
@@ -757,4 +788,64 @@ object Formatter {
     final val AllWrittenFlags =
       LeftAlign | AltFormat | NumericOnlyFlags | UseLastIndex
   }
+
+  private sealed abstract class LocaleInfo {
```

Comment:
This class probably deserves a comment. It is non-obvious that this machinery is necessary to make `Formatter` link without `Locale`.